cfy_cluster_setup/logger.py

```diff
@@ -0,0 +1,25 @@
+import sys
+import logging
+
+
```

Comment:
This section might need a global variable that would signify that a logger was already set-up like we do in `cfy_manager`  https://github.com/cloudify-cosmo/cloudify-manager-install/blob/d418217ba95dfda8ed8bece5efd16c3b2bff4737/cfy_manager/logger.py#L112 
Nevertheless, some of the tests I ran didn't initialize the logger when running a command (`cfy_cluster_setup install`) twice in a row, so I gave it up. Please let me know if I missed something.